product_attribute_auto_add/models/product_attribute.py

```diff
@@ -0,0 +1,17 @@
+# Copyright Cetmix OU 2025
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+
+from odoo import fields, models
+
+
+class ProductAttribute(models.Model):
+    _inherit = "product.attribute"
+
+    attribute_line_auto_add = fields.Boolean(
+        string="Auto-Create Variants",
```

Comment:
I think this string is not correct as it won't create variant automatically (as it depends on `create_variant` configuration - https://github.com/odoo/odoo/blob/17.0/addons/product/models/product_attribute.py#L23)